### ISSUE TITLE
Manual state.json submission after DCS crash without needing file edit to flip flag

### ISSUE THREAD

**Issue by Starfire13** [dcs-liberation/dcs_liberation#2605]
**Is your feature request related to a problem? Please describe.**

At the moment, it appears that users have to edit the state.json file to change "mission_ended": false to true before manually submitting if DCS crashes.

It would be a convenience if Liberation either disregards that flag for manual submissions or automatically flips it to true before processing the json data.

**Comment by DanAlbert** [comment]
> At the moment, it appears that users have to edit the state.json file to change "mission_ended": false to true before manually submitting if DCS crashes.

Or what?

> It would be a convenience if Liberation either disregards that flag for manual submission

IME it does

**Comment by Starfire13** [comment]
> > At the moment, it appears that users have to edit the state.json file to change "mission_ended": false to true before manually submitting if DCS crashes.
> 
> Or what?

You go right back to the Waiting for Mission Completion screen

> > It would be a convenience if Liberation either disregards that flag for manual submission
> 
> IME it does

That's not what I'm seeing. I set it to false, manual submit doesn't work. Set it true, and it does. Feel free to test using this .miz and state.json.

[Vegas Nerve #4529 Turn 2.zip](https://github.com/dcs-liberation/dcs_liberation/files/10121708/Vegas.Nerve.4529.Turn.2.zip)

**Comment by RndName** [comment]
I've found out that this is is a bug which seems to be there for about 13 months already https://github.com/dcs-liberation/dcs_liberation/commit/b728fcc2d6877b61a285cd3c360cfde43cde14dc#diff-2f73c18705a2b3add15147cd68233a86b9bd3053271d2cf93a5e392d245a5ebdR234 Really small one as there is just a bracket which is misplaced. Already about to create the PR for it.
I've debugged that part a lot as i fixed a bug with an open thread which prevented liberation exe from being closed. Strange that i have not found that one earlier
